code/game/gamemodes/cult/runes/stun.dm

```diff
@@ -20,6 +20,8 @@
 				C.stuttering = 1
 			if(is_rune)
 				C.Weaken(3)
+			C.make_dizzy(60)
```

Comment:
This does not actually do anything. It has to be over 100 for dizziness to process. Imho the stun runes don't need dizzy.